engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreHeartbeatWriteActor.scala

```diff
@@ -64,7 +100,7 @@ case class WorkflowStoreHeartbeatWriteActor(workflowStoreAccess: WorkflowStoreAc
   We are expecting the underlying FSM to ensure that the call to this method does NOT occur in parallel, waiting for
   the call to `process` to complete.
    */
-  private def trackRepeatedFailures(heartbeatDateTime: OffsetDateTime, workflowCount: Int)(processTry: Try[Int]): Unit = {
+  private def trackRepeatedFailures(heartbeatDateTime: OffsetDateTime)(processTry: Try[Int]): Unit = {
```

Comment:
I wonder if `workflowCount` was intended to be used on line 124 (instead of `failureShutdownDuration`). Still not sure that the logged error message makes perfect sense, but I think workflow count makes more sense than the failure shutdown duration.

Reply:
ooh yes, great catch!